contrib/udfs/pom.xml

```diff
@@ -66,7 +66,7 @@
     <dependency>
       <groupId>nl.basjes.parse.useragent</groupId>
       <artifactId>yauaa</artifactId>
-      <version>5.11</version>
+      <version>5.13</version>
```

Comment:
There is `5.15` version of this lib in maven central, should we use it?

Reply:
Unfortunately versions 5.14 and 5.15 use a banned dependency which causes problems when you exclude it.  I'm going to work with the developer to see if we can fix that.